volume/drivers/nfs/nfs.go

```diff
@@ -285,31 +292,121 @@ func (d *driver) Create(
 		}
 	}
 
-	f, err := os.Create(path.Join(volPathParent, volumeID+nfsBlockFile))
-	if err != nil {
-		logrus.Println(err)
-		return "", err
-	}
-	defer f.Close()
+	// Create volume
+	var v *api.Volume
+	if spec.GetShared() {
+		// File based
+		v = common.NewVolume(
+			volumeID,
+			api.FSType_FS_TYPE_NFS,
+			locator,
+			source,
+			spec,
+		)
+
+		// Create a directory on the NFS server with this UUID.
+		err := os.MkdirAll(volPath, 0744)
```

Comment:
I've seen 0750 used for directory permissions in k8s.. not sure why above 744 but just a thought:
https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/csi/csi_block.go#L166

Reply:
great point!